Pract08_06/Task3/TextCount.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace Pract08_06.Task3
+{
+    class TextCount
+    {
+        public void CountText()
+        {
+            var Path = @"C:\FolderForCurces\Text.txt";
+            var file = File.ReadAllText(Path);
+            var counter = 0;
+
+            for (int i = 0; i < file.Length; i++)
+            {
+                if (file[i] != ' ' && file[i] != '\n')
+                {
```

Comment:
Why did you leave brackets in 'if' statement and removed them from 'else' statement?